BraintreeCore/src/main/java/com/braintreepayments/api/ConfigurationLoader.java

```diff
@@ -36,20 +36,39 @@ void loadConfiguration(final Context context, final Authorization authorization,
                 .build()
                 .toString();
 
-        Configuration cachedConfig = getCachedConfiguration(context, authorization, configUrl);
+
+        Configuration cachedConfig = null;
+        BraintreeSharedPreferencesException loadFromCacheException = null;
+        try {
+            cachedConfig = getCachedConfiguration(authorization, configUrl);
+        } catch (BraintreeSharedPreferencesException e) {
+            loadFromCacheException = e;
+        }
+
```

Comment:
Instead of declaring variables and exceptions and then check which one is null, can we simply execute the succesfull flow inside the `try` and the error flow inside `catch`? Something like:
```java
        try {
            Configuration cachedConfig = getCachedConfiguration(authorization, configUrl);
            // return config to callback
        } catch (BraintreeSharedPreferencesException e) {
            BraintreeSharedPreferencesException loadFromCacheException = e;
            // execute getting configuration remotley
        }
```